modules/ROOT/pages/audit-logging.adoc

```diff
@@ -2099,6 +2099,127 @@ You can access the
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/[Audit Log REST API] from the Audit Logging Query API and its
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/organizations/8bfc8bbf-5508-419e-aadc-77dfe18a8172/assets/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/1.0.5/files/fat-raml/zip[RAML].
 
+=== Use a Query Loop to make Requests in Audit Logs
```

Comment:
Hi Sara, I used your PR as base and created a new branch and added more content since I thought that would be easier to communicate the content that I think should be included;
Please see that here:
https://github.com/mulesoft/docs-access-management/commit/8a2366f15768476d02a039d647dc9b47db9374a5
Can you please treat this as my spec and work your magic to document it in a way that is understandable :) thanks for your patience and for your help to document this!